_includes/com-install/com-install-startup-fb.md

```diff
@@ -0,0 +1,6 @@
+* Open a terminal and CD to the `/featurebase/opt` directory.
```

Comment:
You don't CD to a location. You CD to a directory. 

Alternatively, the CD isn't strictly relevant because if someone's using the system then they'll know that kind of stuff:

```
* Open a Terminal then the `/featurebase/opt` directory.
```

Of course that'll mean I need to change it across the board. But it won't be hard. Find and replace works just fine. I mean, what could possibly go wrong :D